pages/checkIn.tsx

```diff
@@ -1,5 +1,44 @@
+import CheckInOutForm from 'components/CheckInOutForm'
+import {
+  Box,
+  Button,
+  Card,
+  CardContent,
+  CardActions,
+  Typography,
+  useMediaQuery,
+} from '@mui/material'
+
 export default function CheckInPage() {
+  const isMobileView = useMediaQuery('(max-width: 600px)')
   return (
-    <h1>Check In</h1>
+    <Box
+      mx="auto"
+      display="flex"
+      flexDirection="column"
+      maxWidth="min(600px, 80%)"
+    >
+      <Button variant="outlined" sx={{ alignSelf: {xs: 'center', sm: 'end'} }}>
+        Create new item
+      </Button>
+      <Card variant={isMobileView ? 'elevation' : 'outlined'} elevation={0}>
+        <Box px="1.5rem" py="2rem" display="flex" flexDirection="column">
```

Comment:
remove `px` and `py` and instead use `p={2}` I don't think its necessary to use different padding for x and y